resources/assets/js/annotations/ol/MagicWandInteraction.js

```diff
@@ -188,6 +187,9 @@ class MagicWandInteraction extends PointerInteraction {
             this.sketchSource.removeFeature(this.sketchFeature);
         } else {
             this.dispatchEvent({type: 'drawend', feature: this.sketchFeature});
```

Comment:
I was wondering why there was a delay between the new feature in it's "editing" state (i.e. dashed line) and the "created" state (i.e. solid line) after finishing with the magic wand. The reason is that the interaction sets the editing style directly on the feature and this is moved over to the annotation source, so it keeps the style after `drawend`. Only when the HTTP request returned, the feature is recreated with the proper style.

To fix this, please add `this.sketchFeature.setStyle(null)` above `dspatchEvent()`, so the feature is passed to the event without the editing style.